dev/TeachingTip/TeachingTip.cpp

```diff
@@ -171,10 +173,45 @@ void TeachingTip::OnPropertyChanged(const winrt::DependencyPropertyChangedEventA
     else if (property == s_TitleProperty)
     {
         SetPopupAutomationProperties();
+        if (ToggleVisibilityForEmptyContent(m_titleTextBox.get(), Title()))
+        {
+            TeachingTipTestHooks::NotifyTitleVisibilityChanged(*this);
```

Comment:
Nit: This is ok, but from the test you could reach into the template, find the textBox and hook up to its property changed event also.